tide_predictions/README.md

```diff
@@ -0,0 +1,43 @@
+# Tide Prediction
+
+This directory contains code to predict tide for NOAA gauge stations when given a station ID and date(s) of prediction.
+
+Method used to predict tides in this example is adapated from Pytides, a python package developed by Sam Cox. The method used is that of harmonic constituents, in particular as presented by P. Schureman in Special Publication 98.
+
+Harmonic Constituents data is scraped from NOAA gauge station harmonic constituents page. 
+
+# Requirements:
+
+* Numpy 
+* Scipy
+* Matplotlib
+* Pandas 
+* Request 
+* lxml
+
+# Installations:
+
+```
+pip install numpy
+pip install scipy
+pip install matplotlib
+pip install pandas
+pip install requests
+pip install lxml
+```
+
```

Comment:
We could actually specify this as a `requirement.txt` file.  Not sure that it should not be here though still.  Probably do not need to include the `pip install` part thought